.env

```diff
@@ -1 +1,7 @@
-DATABASE_URL=postgres://openadr@localhost/openadr
\ No newline at end of file
+DATABASE_URL=postgres://openadr:openadr@localhost:5433/openadr
+VTN_PORT=3000
+PG_PORT=5433
```

Comment:
I would prefer to keep the default Pg port in the default setup. Feel free to keep it changed in your local environment. (Also affects line 1)